vars/main-sriov.yml

```diff
@@ -0,0 +1,87 @@
+k8s:
+  rke2:
+    version: v1.23.4+rke2r1
+    config:
+      token: purdue-k8s-rke2
+      port: 9345
+
+  kubectl:
+    version: v1.23.15
+
+  helm:
+    version: v3.10.3
+
+core:
+  standalone: "true"
+  data_iface: ens18
+  # If upf.mode set to 'dpdk' then use below file for 'values_file',
+  # "deps/5gc/roles/core/templates/sdcore-5g-sriov-values.yaml"
+  values_file: "deps/5gc/roles/core/templates/sdcore-5g-sriov-values.yaml"
+  ran_subnet: "172.20.0.0/16"	# set to empty string to get subnet from 'data_iface'
```

Comment:
@sureshmarikkannu, let's have a quick discussion about this. I thought this should be empty (`""`) when deploying the UPF in DPDK mode

Reply:
i did verified by configuring specific subnet and keep it as empty. In both cases all works fine.